package.json

```diff
@@ -38,6 +38,7 @@
   "dependencies": {
     "@gnosis.pm/gp-v2-contracts": "^1.1.2",
     "ajv": "^8.8.2",
+    "cids": "^1.1.9",
```

Comment:
Thinking out loud as we are currently working on something similar on CowSwap

Would it make sense to mark it as a peer dependency and only load it if actually dealing with data decoding?

I guess in that case it would have to be exposed as a different module?

I'm not really aware of how to make tree-shakable libs.

Anyway, might make sense as a new task

Reply:
I'm not sure if in this case is a peer dependency, or it would be more just making sure is tree-shakable if we don't use the IPFS functions (which many people won't use)